src/semgrep_agent/main.py

```diff
@@ -87,13 +88,25 @@ def main(
     # Setup URL/Token
     sapp = Sapp(url=publish_url, token=publish_token, deployment_id=publish_deployment)
     maybe_print_debug_info(meta)
-    sapp.report_start(meta)
+    policy = sapp.report_start(meta)
     if sapp.is_configured:
         click.echo(
-            f"| semgrep.dev - logged in as deployment #{sapp.deployment_id}", err=True,
+            f"| {server} - logged in as deployment #{sapp.deployment_id}", err=True,
```

Comment:
```suggestion
            f"| Semgrep Community - logged in as deployment #{sapp.deployment_id}", err=True,
```

I'd conditionally add the URL, only if it's not the default. Something like:

```
logged in to dev.semgrep.dev as deployment #1
```